examples/high_level_plots.py

```diff
@@ -71,19 +71,26 @@
 results.plot_var_perf(
     working_point=0.7,
     bins=[20, 30, 40, 60, 85, 110, 140, 175, 250],
-    fixed_eff_bin=False,
+    flat_eff_bin=False,
```

Comment:
What do you think about changing this from `flat_eff_bin` to `flat_per_bin`, since sometimes it's a rejection that we hold flat?